setup.py

```diff
@@ -1072,6 +1072,54 @@ def make_relative_rpath_args(path):
     return extensions, cmdclass, packages, entry_points, extra_install_requires
 
 
+def add_triton(install_requires, extras_require) -> None:
```

Comment:
After this, we would only need the change on builder, i.e. https://github.com/pytorch/builder/pull/1515, instead of changing triton version in two different places.